ui/components/FilterDialog.tsx

```diff
@@ -78,20 +78,19 @@ const FilterSection = ({
           </Text>
         </ListItem>
         {options.sort().map((option: string, index: number) => {
```

Comment:
You can use a shorter form here, because you are only returning the ListItem without using any local variables here:

```
{options.sort().map((option: string, index: number) => (
          <ListItem key={index}>
            <ListItemIcon>
              <FormCheckbox
                label=""
                name={`${header}${filterSeparator}${option}`}
              />
            </ListItemIcon>
            <Text color="neutral40" size="small">
              {_.toString(option) || "-"}
            </Text>
          </ListItem>
        ))}
```